Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -0,0 +1,129 @@
+        });
+      });
+    });
+
+    it("debugShowBoundingVolume works", function () {
+      var resource = Resource.createIfNeeded(boxTexturedGlbUrl);
+      var loadPromise = resource.fetchArrayBuffer();
+      return loadPromise.then(function (buffer) {
+        return loadAndZoomToModelExperimental(
+          { gltf: new Uint8Array(buffer), debugShowBoundingVolume: true },
+          scene
+        ).then(function (model) {
+          var i;
+          scene.renderForSpecs();
+          var commandList = scene.frameState;
+          for (i = 0; i < commandList.length; i++) {
+            expect(commandList[i].debugShowBoundingVolume).toBe(true);
+          }
+          model.debugShowBoundingVolume = false;
+          expect(model._debugShowBoundingVolumeDirty).toBe(true);
+          scene.renderForSpecs();
+          for (i = 0; i < commandList.length; i++) {
+            expect(commandList[i].debugShowBoundingVolume).toBe(false);
+          }
+        });
+      });
+    });
+
+    it("boundingSphere works", function () {
+      var resource = Resource.createIfNeeded(boxTexturedGlbUrl);
+      var loadPromise = resource.fetchArrayBuffer();
+      return loadPromise.then(function (buffer) {
+        return loadAndZoomToModelExperimental(
+          { gltf: new Uint8Array(buffer), debugShowBoundingVolume: true },
+          scene
+        ).then(function (model) {
+          var boundingSphere = model.boundingSphere;
+          expect(boundingSphere).toBeDefined();
+          expect(boundingSphere.center).toEqual(new Cartesian3());
+          expect(boundingSphere.radius).toEqualEpsilon(
+            0.8660254037844386,
+            CesiumMath.EPSILON8
+          );
+        });
+      });
+    });
+
+    it("fromGltf throws with undefined options", function () {
+      expect(function () {
+        ModelExperimental.fromGltf();
+      }).toThrowDeveloperError();
+    });
+
+    it("fromGltf throws with undefined url", function () {
+      expect(function () {
+        ModelExperimental.fromGltf({});
+      }).toThrowDeveloperError();
+    });
+
+    it("destroy works", function () {
+      var resource = Resource.createIfNeeded(boxTexturedGlbUrl);
+      var loadPromise = resource.fetchArrayBuffer();
+      return loadPromise.then(function (buffer) {
+        return loadAndZoomToModelExperimental(
+          { gltf: new Uint8Array(buffer) },
+          scene
+        ).then(function (model) {
+          var buffer = Buffer.createVertexBuffer({
+            context: scene.frameState.context,
+            sizeInBytes: 16,
+            usage: BufferUsage.STATIC_DRAW,
+          });
+          model._resources = [buffer];
+
+          expect(buffer.isDestroyed()).toEqual(false);
+          expect(model.isDestroyed()).toEqual(false);
+          scene.primitives.remove(model);
+          expect(buffer.isDestroyed()).toEqual(true);
+          expect(model.isDestroyed()).toEqual(true);
+        });
+      });
+    });
```

Comment:
It would be good to check that the resources held by `GltfLoader` are also destroyed.

And another test that loads two of the same models, deletes one, and checks that the resources held by `GltfLoader` are not destroyed, i.e. because the reference counts never hit zero.